wgpu-types/src/lib.rs

```diff
@@ -1112,12 +1496,19 @@ impl Default for AddressMode {
     }
 }
 
+/// Texel mixing mode when sampling between texels.
 #[repr(C)]
 #[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
 #[cfg_attr(feature = "trace", derive(Serialize))]
 #[cfg_attr(feature = "replay", derive(Deserialize))]
 pub enum FilterMode {
+    /// Nearest neighbor sampling.
+    ///
+    /// This creates a pixelated effect when used as a mag filter
     Nearest = 0,
+    /// Linear Interpolation
+    ///
+    /// This makes textures smooth but blurry when used as a mag filter.
```

Comment:
why are we only specifying this for mag filters?

Reply:
Because I didn't know how it behaved with in filters and explaining it for mip filters is hard.